src/components/core/scrollbars/scrollbars.tsx

```diff
@@ -100,7 +100,7 @@ const Scrollbars = React.forwardRef(
         const [is_mouse_down, setIsMouseDown] = React.useState(false);
         const [start_clientY, setStartClientY] = React.useState<number | null>(null);
         const [current_clientY, setCurrentClientY] = React.useState<number | null>(null);
-
+        const { isDark } = useTheme();
```

Comment:
```suggestion
        const { isDark } = useTheme();
```
Nit, can add a space under this to group hooks and handlers? 🙏🏼